product_docs/docs/pgd/5/upgrades/upgrade_paths.mdx

```diff
@@ -2,28 +2,26 @@
 title: Supported PGD upgrade paths
 ---
 
-
 ## Upgrading within version 5
 
-| 5.0.0 | 5.0.1 | 5.1.0 | Target PGD version |
-|-------|-------|-------|--------------------|
-| ✓     | ✓     | ✓     | 5.2.0              |
-| ✓     | ✓     |       | 5.1.0              |
-| ✓     |       |       | 5.0.1              |
-
+You can upgrade from any version 5.x release to a later 5.x release. 
 
 ## Upgrading from version 4 to version 5
 
-Upgrades from PGD 4 to PGD 5 are supported from version 4.3.0. For older versions, upgrade to 4.3.0 before upgrading to 5. See [Upgrading within 4](/pgd/4/upgrades/upgrade_paths/#upgrading-within-version-4) for more information. After upgrading to 4.3.0 or later, the following combinations are allowed.
-
-| 4.3.0 | 4.3.0-1 | 4.3.1 | 4.3.1-1 | Target PGD version |
-|-------|---------|-------|---------|--------------------|
-| ✓     |  ✓      |   ✓   |    ✓    | 5.2.0              |
-| ✓     |  ✓      |   ✓   |         | 5.1.0              |
-| ✓     |  ✓      |       |         | 5.0.1              |
-| ✓     |         |       |         | 5.0.0              |
-
-
+Generally, we recommend you upgrade to the latest version 4 release, before
+upgrading to the latest version 5 release. Upgrades from PGD 4 to PGD 5 are
+ supported from version 4.3.0 or later. See [Upgrading within
+4](/pgd/4/upgrades/upgrade_paths/#upgrading-within-version-4) for more
+information. After upgrading t the following upgrade paths are
+possible.
+
+| From version | To version |
+| ---- | -- |
+| 4.3.0 | 5.0.0 or later |
+| 4.3.0-1 | 5.0.1 or later |
```

Comment:
I'm still not quite clear what `4.3.0-1` refers to. In the packaging repo, I see all of these tags:
* `REL_4_3_0-1`
* `REL_4_3_0-2`
* `REL_4_3_0-3`

...there's no `REL_4_3_0-0`, though. So ... what's the difference between `4.3.0` vs `4.3.1-1`?

Clearly, BDR version 5.0.1 provides an upgrade script from 4.3.0, so for that reason I'd conclude that "4.3.0" (any packaging revision) is upgradable to "5.0.1 or later" (but then scratch any "4.3.0-1" rows).

Reply:
Okay, that got clarified via https://github.com/EnterpriseDB/docs/pull/4755. That however means all the `+p1` versions match their original one, because the only component that really matters for upgrades is BDR.

Reply:
So for the entry 

| 4.3.1 | 5.1.0 or later |
| 4.3.1+p1 | 5.2.0 or later |
| 4.3.2 | 5.2.0 or later |

Do you want the +p1 row there deleted? or retained? Or is that another error in the upgrade data?